met-web/src/components/engagement/admin/view/index.tsx

```diff
@@ -32,6 +42,52 @@ export const AdminEngagementView = () => {
                     )}
                 </Await>
             </Suspense>
-        </div>
+            <TabContext value={currentTab}>
+                <TabList
```

Comment:
Does this render as a `nav`? If not, we can specify the component element to use, or give it an `aria-role` of `navigation`

Reply:
It renders as a div right now. I'll see if I can make it render as a nav instead.

Reply:
Thanks!

Reply:
Rendering as a nav element as of the latest commit